pillar/config.sls

```diff
@@ -1,3 +1,5 @@
 hostname: raspberrypi
 domain: moio
 # additional_search_domains: mycompany.example, myothercompany.example
+# user_name: iceman
```

Comment:
Seems like the default is `user`, shouldn't this be reflected here?

Reply:
Yes, let's use `user`